products/memgraph/upcoming/tutorials/marvel-universe.md

```diff
@@ -0,0 +1,165 @@
+## Marvel Universe Social Network
+
```

Comment:
There is trailing whitespace in the document that should be removed (lines 50, 84).

Reply:
Indeed.